msteams-platform/bots/how-to/format-ai-bot-messages.md

```diff
@@ -0,0 +1,401 @@
+
+## AI label
+
+AI label added to your bot message specifies that the message is generated by AI. AI bots use large language models (LLMs) that are generally reliable, however, adding an AI label prevents confusion about the source of the information.
+
+For a bot built using Teams AI library, an AI label is automatically enabled for all AI-powered bot messages in the `ai` module within the `PredictedSayCommand` action. For more information, see [Teams AI code sample](https://github.com/microsoft/teams-ai/blob/main/js/packages/teams-ai/src/actions/SayCommand.ts).
+
+If you're using Bot Framework SDK to build your bot, include `additionalType` under `entities` array of the `message` entity object. Here's a code snippet for your reference:
+
+```javascript
+await context.sendActivity({
+         type: ActivityTypes.Message,
+         text: `Hey! I'm a friendly AI bot. This message is generated by AI. [1]`, // cite with [1],
+        entities: [
+          {
+            type: "https://schema.org/Message",
+            "@type": "Message",
+            "@context": "https://schema.org",
+            additionalType: ["AIGeneratedContent"], // AI Generated label
+          }
+        ]
+       });
+```
+
+| Property | Type | Required | Description |
+|--|--|--|--|
+| `additionalType` | String | Yes | Enables the AI label in the bot message |
+
+After an AI label is added, your bot’s message displays an **AI generated** label next to the bot’s name with a hover-over disclaimer stating, **AI-generated content may be incorrect**. The AI label and disclaimer can't be customized for AI-powered bots.
+
+:::image type="content" source="../../assets/images/bots/ai-bot-label.png" border="false" alt-text="Screenshot shows an AI label in a bot message.":::
+
+## Citations
+
+It's important to cite the sources of the bot message to help users ask follow-up questions or conduct independent research. Cite data sources, such as files, messages, emails, and work items, to provide valuable insights to users. Citations are important for bots using techniques such as Retrieval Augmented Generation (RAG).
+
+Citations to your bot message include in-text citations, details to citation reference, and sensitivity labels for the content referenced.
+
+* **In-text citations** denote the citation numbers added to the bot message in the [#] format, each corresponding to a reference. A citation can be inserted anywhere within the text.
+* **Details to citation reference** include the title, keywords, an abstract, hyperlink, and sensitivity information. References appear as pop-ups for each in-text citation.
+* **Sensitivity labels to citations** indicate the confidentiality of the citation content referenced and aren't added automatically. To add sensitivity labels for citations, see [modify PredictedSayCommand](#modify-predictedsaycommand).
+
+> [!NOTE]
+>
+> * You can't add more than 10 citations per message.
```

Comment:
The next statement says that one can add more citations in the 'entities' field which contradicts with this statement. Since, we're not going to display citation references in the message footer, change this statement to something along the lines of 'A maximum of only 10 citations are displayed in the message'

Reply:
Updated as suggested.